_data/retail.yml

```diff
@@ -55,6 +55,13 @@ websites:
       facebook: Arcaplanet
       lang: it
 
+    - name: Best Buy
+      url: https://www.bestbuy.com
+      img: bestbuy.png
+      tfa: No
+      twitter: BestBuy
```

Comment:
```suggestion
      twitter: BestBuySupport
```

Reply:
![bestbuy](https://user-images.githubusercontent.com/5554308/65808174-84e46900-e147-11e9-82dc-3763b09c6132.png)


How about this with the blue background?